### DISCUSSION TITLE
URL's do not get redirected (ERR_CONNECTION_REFUSED)

### DISCUSSION THREAD

**Discussion by h-enk** [codesandbox/codesandbox-client#8536] [Q&A] [answered]
The main preview URL (https://ptjs7z-1313.csb.app/) does work, but `localhost:1313` URL's do not get redirected (ERR_CONNECTION_REFUSED) — click for example on the "Example Guide" button.

What am I missing? Thanks for your help!

https://codesandbox.io/p/devbox/github/gethyas/doks-codesandbox

**Comment by filipelima18** [comment, marked as answer]
Hi @h-enk. The URL should point to relative paths like `/docs/guides/example-guide/`.

**Reply by h-enk** [reply]
Hi @filipelima18 , thanks for pointing me in the right direction!

I got it working using (all arguments are needed):

```bash
hugo server --baseURL=/ --appendPort=false --liveReloadPort=443 --bind=0.0.0.0
```

**Reply by filipelima18** [reply]
Great to hear you solved it! And thanks for leaving the solution that worked for you, so others can reference it if needed.
